src/core/Akka/Akka.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+      <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="5.0.5" />
```

Comment:
At some point this will get folded into the core runtime when we move on from .NET Standard 2.0, IIRC